app/main.py

```
@@ -301,6 +320,41 @@ def is_early_access(year):
+        dict: 共通変数
+    """
+    # 年度が公開範囲内か検証
+    year_str = g.current_url.split("/")[1]
+    is_latest_year = None
+    is_early_access = None
+
+    # 年度が最新 or 試験公開年度か検証
+    try:
+        year = int(year_str)
+        if year in AVAILABLE_YEARS:
+            is_latest_year = is_latest_year(year)
```

Pylint (3.3.7)

[error] is_latest_year is not callable

(E1102)